README.md

```diff
@@ -152,7 +152,7 @@ These conversations include all conversations for a user **regardless of which a
 You can also listen for new conversations being started in real-time. This will allow applications to display incoming messages from new contacts.
 
 > **Warning**  
-> This stream will continue infinitely. To end the stream you can call `cancelStream()`.
+> This stream will continue infinitely. To end the stream you can call the method returned by `conversations.stream()`.
 
 ```tsx
 const stream = await xmtp.conversations.stream()
```

Comment:
maybe we can update this code example to more clearly show how to cancelStream using the function returned by 
`xmtp.conversations.stream()`

At the very least we can update it to something like

`const cancelStream = await xmtp.conversations.stream()`